src/server/cloak-server.spec.js

```diff
@@ -832,7 +828,7 @@ describe('cloak server', () => {
     describe('submitAnswer ', () =>{
         beforeEach( () => {
             room.data = {
-                letterList:{
+                lettersRound:{
```

Comment:
not yet the bulk of the numbers round will be in the next issue and i will add tests for it then